var/spack/repos/builtin/packages/cmake/package.py

```diff
@@ -29,6 +29,7 @@ class Cmake(Package):
     license("BSD-3-Clause")
 
     version("master", branch="master")
+    version("3.28.3", sha256="72b7570e5c8593de6ac4ab433b73eab18c5fb328880460c86ce32608141ad5c1")
```

Comment:
Doesn't this have the same issue as https://github.com/spack/spack/pull/42500 ?

Reply:
Yes it will/currently does, thank you @alalazo.